docs/osquery/osquery.asciidoc

```diff
@@ -120,18 +121,76 @@ image::images/scheduled-pack.png[Shows queries in the pack and details about eac
 . View scheduled query results in <<discover,*Discover*>> or the drag-and-drop <<lens,*Lens*>> editor.
 
 [float]
-[[osquery-prebuilt-packs]]
-== Prebuilt Elastic packs
+[[osquery-manage-query]]
+== Save queries
+
+You can save queries in two ways:
+
+* After running a live query, click the *Save for later* link.
+* From the *Saved queries* tab, click the **Add saved query** button.
+
+Once you save a query, you can only edit it from the *Saved queries* tab.
+
+To add or edit saved queries from the *Saved queries* tab:
+
+. Go to *Saved queries*, and then click **Add saved query** or the edit icon.
+. Provide the following fields:
+
+* The unique identifier.
+
+* A brief description.
+
+* The SQL query.
+
+* The <<osquery-map-fields,ECS fields>> to populate when the query is run. These fields are also copied in when you add this query to a pack.
+
+* The defaults to set when you add the query to a pack.
+
+** The frequency to run the query.
+
+** The minimum https://github.com/osquery/osquery/releases)[version of Osquery] required to run the query.
```

Comment:
This link doesn't work.

Reply:
Good catch, thanks! It looks like there was an extra `(` in there.